tsdb/config.go

```diff
@@ -30,7 +30,7 @@ const (
 
 	// DefaultFlushColdInterval specifies how long after a partition has been cold
 	// for writes that a full flush and compaction are forced
-	DefaultFlushColdInterval = 5 * time.Minute
+	DefaultFlushColdInterval = 5 * time.Second
```

Comment:
yes on purpose, no reason to keep stuff cached in memory for long. If they're only writing once every 5 seconds, they could bypass the wal anyway.